spec/spec_helper.rb

```diff
@@ -1,5 +1,5 @@
-require "codeclimate-test-reporter"
-CodeClimate::TestReporter.start
+require 'simplecov'
+SimpleCov.start
```

Comment:
I agree. However, when I do not add this Github travis will fail since tests refuse to run. Could you add them yourself?